RELEASENOTES.md

```diff
@@ -6,6 +6,8 @@ Releases, starting with 9/2/2021, are listed with the most recent release at the
 __Bug Fixes__:
 
 #1426 Sequential.eval() does not put model into eval mode<br/>
+#1439 `torch.optim.lr_scheduler.LinearLR` `end_factor` default has been corrected, is now 1.0.<br/>
```

Comment:
The number should point to an issue or discussion thread, since this is not one, I say we follow the pattern from here: https://github.com/dotnet/TorchSharp/blob/5ba14022749a20b98798105a32988da59f2356ca/RELEASENOTES.md?plain=1#L287

Reply:
thanks, removed the number.